gatsby-node.js

```diff
@@ -166,7 +166,7 @@ async function runIndexQueries(
         if (queryResultsMap.hasOwnProperty(id)) {
           // key matches fresh objects, so compare match fields
           const newObj = queryResultsMap[id];
-          if (!matchFields.every(field => newObj.hasOwnProperty(field))) {
+          if (!matchFields.some(field => newObj.hasOwnProperty(field))) {
```

Comment:
```suggestion
          if (matchFields.every(field => newObj.hasOwnProperty(field) === false)) {
```

(that's the same as your fix, but it makes the code easier to read IMO)

Reply:
do you mean `.some` instead of `.every`?

Reply:
If every matchField is missing, we expect the condition, right? (Same as !some having the condition, opposite of the existing condition). It'll check this again tomorrow with a fresh mind

Reply:
oh, you're right. I misread and thought the comparator was outside the `every` function 🤦 I'll update the code :)